test/swift/EngineBuilderTests.swift

```diff
@@ -19,6 +19,7 @@ fixture_template:
 
 private struct TestFilter: Filter {}
 
+// swiftlint:disable:next type_body_length
```

Comment:
not sure if this is the kosher way of doing this @rebello95

Reply:
this is fine, or honestly we could just remove this rule from `.swiftlint.yml`

Reply:
the "fix" would be to split this file up